### ISSUE TITLE
Background image tricks axe

### ISSUE THREAD

**Issue by adriantoddross** [adriantoddross/dreadakidd#7]
The way I've implemented the background image tricks a11y testing tools into thinking all elements on the page are contrasted against a black background, but I'm actually using a gradient as a fallback and an image primarily.
